config/ember-try.js

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 module.exports = async function () {
   return {
     useYarn: true,
     scenarios: [
+      embroiderSafe(),
+      embroiderOptimized(),
       {
         name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
             'ember-source': '~3.28.0',
-            'ember-qunit': '^5.0.0',
-            '@ember/test-helpers': '^2.4.0',
```

Comment:
Technically ember-qunit @ latest wants newer ember-source than 3.28 (via peerDeps), but the CI still passes if we don't explicitly downgrade it and ember/test-helpers for this scenario, so I removed them to make the scenarios simpler.